source/state_representation/include/state_representation/space/cartesian/CartesianPose.hpp

```diff
@@ -148,7 +150,7 @@ class CartesianPose : public CartesianState {
    * @param pose CartesianPose to multiply with
    * @return the current CartesianPose multiplied by the CartesianPose given in argument
    */
-  CartesianPose& operator*=(const CartesianPose& pose);
+  [[deprecated]] CartesianPose& operator*=(const CartesianPose& pose);
```

Comment:
These pose *= pose and state *= state operators are the ones that actually do make sense, as you mentioned in your comment - so why have you deprecated it here?

Reply:
If I undersrand correctly we have two choices for the *= operator. Either we delete it completely or we have it in CartesianState, override it in CartesianPose and delete it in Wrench and Twist. And i was going for the first option hehe

Reply:
well I would prefer the second option though `*=` for pose and state make sense so I would keep them and just delete them in the velocity and wrench

Reply:
Yeah but let's face it then, either we have breaking changes or we postpone the *= operator until the next major release. Otherwise I have to implement every combination and mark them as deprecated instead of just deleting them where I can